### PR TITLE
net-im/skypeforlinux: add 8.92.0.401

### PR DESCRIPTION
**Pull request by rahul-singh-sandhu**
Signed-off-by: Rahul Sandhu <rahul@sandhuservices.dev>

May I also request to be added as a maintainer for this package? This is my second pr for a version bump, and as I use it myself I would like to maintain it.

### CODE REVIEW

File: net-im/skypeforlinux/Manifest [hunk 1]
```diff
@@ -1,2 +1,7 @@
...
+EBUILD skypeforlinux-8.91.0.404.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+EBUILD skypeforlinux-8.92.0.401.ebuild 3250 BLAKE2B eed5437754666a9da47304ab1da0539bb8e3e0e7f7e462b7e0de527038cc024f7f0d3c513ffcaaacac9d4c3223de6aa07b26f14f00d2bfa5b9ee872857909eb2 SHA512 09758ed08729dbb173e6d03fc0dbc1e04f3271ba42ff1e837d73bddb88a07a55fe67d64bfd59986a3af67e39d11e170459a2e92e1cf51f7f911cbc3f6b3bf55e
+MISC metadata.xml 461 BLAKE2B 9bcbfe1d6892c40ce917a76c600baad338b5b86967e5656f79b9bf29e75c3ba474dc11444d8e347769f37934610a971097f96b22ae171ea46cd3f1aef2e9f3e5 SHA512 81df3ef175eafabad357f3d5792c71534b85ffd2c1bd9280ea7ef9cbea17aceb396bc7ac65abdb09e5fba8c3d98384c0d67c58a8daa2d42a657e1a7b150b52e7
```
Comment (ajakk) [inline net-im/skypeforlinux/Manifest:7]: How did these get here? You need [thin manifests] for gentoo.git, and I'm not sure if `pkgdev manifest` would even emit this.

**Review by ajakk — CHANGES_REQUESTED** [review]

File: net-im/skypeforlinux/Manifest (hunk 1 above)
Comment (rahul-singh-sandhu) [reply net-im/skypeforlinux/Manifest:7]: Oh okay - I'll generate a thin manifest when I get back later today. Thanks ajak

**Comment by ajakk** [comment]
Ping?

**Comment by rahul-singh-sandhu** [comment]
Pong. Sorry I’m not at my house at the moment, so unfortunately I do not have a gentoo/linux box on me and so I’ll need to generate them in a bit. I’ll probably generate the thin manifests on Saturday/Sunday when I return home and have access to my box.

**Comment by SoapGentoo** [comment]
updated

**Pull request closed without merging**